test/health_checker_unittest.cc

```diff
@@ -8,36 +8,23 @@
 namespace google {
 
 class HealthCheckerUnittest : public ::testing::Test {
- public:
-  HealthCheckerUnittest() { }
-
+ protected:
   ~HealthCheckerUnittest() {
     if (isolation_path_.length() > 0) {
       std::ostringstream oss;
-      oss << isolation_path_ << "/metadata_agent_unhealthy";
+      oss << isolation_path_ << "/unhealthy";
```

Comment:
How about we just add a private `Cleanup` function to `HealthChecker` to do this?

Reply:
Hmm, I suppose we could, though I think it would be confusing to include a cleanup that is not supposed to be called (except in tests).

Reply:
As discussed offline, this function would only be of use in tests, so let's make that explicit by calling it `CleanupForTest`.

Reply:
Done.